aesthetic/add_autotags.py

```diff
@@ -53,7 +53,6 @@ def process_some_rows(start_point, end_point, photos):
     print "%s\tStarting." % time.asctime()
     city_csv_reader = csv.reader(open(args.city_csv_file))
     photos = {int(row[0]): row for row in city_csv_reader}
-
```

Comment:
hey! if you've just got whitespace changes, don't bother committing it. You can do this by:

    git checkout HEAD -- aesthetic/add_autotags.py

which will reset add_autotags.py to whatever it was in HEAD. Yes, that is a confusing use of "checkout" because usually "checkout" is used for other things, but just think of it as a separate command - in order to put something back to the way it was in HEAD (that is, before you did anything), you can do "git checkout HEAD -- (filename)".